osu.Game/Overlays/Mods/ModColumn.cs

```diff
@@ -67,8 +67,25 @@ public IReadOnlyList<ModState> AvailableMods
         private IModHotkeyHandler hotkeyHandler = null!;
 
         private Task? latestLoadTask;
-        private ICollection<ModPanel>? latestLoadedPanels;
-        internal bool ItemsLoaded => latestLoadTask?.IsCompleted == true && latestLoadedPanels?.All(panel => panel.Parent != null) == true;
+        private ModPanel[]? latestLoadedPanels;
+        internal bool ItemsLoaded => latestLoadTask?.IsCompleted == true && allPanelsLoaded;
```

Comment:
After switching to array, you should be able to use static predicate without allocs and avoid yawn methods.

```suggestion
        internal bool ItemsLoaded => latestLoadTask?.IsCompleted == true && latestLoadedPanels?.All(static panel => panel.Parent != null) == true;
```

Reply:
Well...
![Снимок экрана 2024-02-26 203100](https://github.com/ppy/osu/assets/22874522/fda1a522-0c7f-4f8f-8314-5961cba07e8a)